physics/smoke_dust/rrfs_smoke_wrapper.F90

```diff
@@ -941,7 +951,7 @@ subroutine rrfs_smoke_prep(                                               &
 
 !---- Calculate HWP based on selected method
     hwp_local = 0._kind_phys
-    precip_factor  = 5._kind_phys + real(hour_int)*5._kind_phys/24._kind_phys
+    precip_factor  = 2.5_kind_phys + real(hour_int)*5._kind_phys/24._kind_phys
```

Comment:
Change `real(hour_int)` to `real(hour_int, type=kind_phys)`

Reply:
@dustinswales I updated the PR with above. However, the compilation failed as below. Did I miss something? Thanks.

/scratch2/BMC/acomp/Haiqin.Li/Commit/RRFS.v1/ufs-weather-model/FV3/ccpp/physics/physics/smoke_dust/rrfs_smoke_wrapper.F90:954:41:

  954 |     precip_factor  = 2.5_kind_phys + real(hour_int, type=kind_phys)*5._kind_phys/24._kind_phys
      |                                         1
Error: Cannot find keyword named 'type' in call to 'real' at (1)

Reply:
Sorry, typo on my end. It should be "kind" not "type".

Reply:
@dustinswales Thanks. I will update accordingly.